api/core/workflow/nodes/agent/entities.py

```diff
@@ -24,3 +24,18 @@ class AgentInput(BaseModel):
 class ParamsAutoGenerated(Enum):
     CLOSE = 0
     OPEN = 1
+
+
+class AgentOldVersionModelFeatures(Enum):
```

Comment:
Is `AgentOldVersionModelFeatures` essentially a duplicate of `ModelFeatures` without the `STRUCTURED_OUTPUT` type?

Duplicating `ModelFeatures` and renaming it as `OldVersion` seems like a questionable approach, as it can complicate long-term code maintenance.